.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,78 @@
+name: Integration Tests
```

Comment:
Question(s) maybe for @vishalkharge 
- This isn't running on a docker image, is it? It's running on GH runners, so should we be mindful of the number of tests and the number of re-tries we perform?